### PR TITLE
[master < ] Add migration from Neo4j tutorial 

### PR DESCRIPTION
**Pull request by antejavor**
[master < 801-formatting-email-username-when-creating-an-user] Update docs with new info

### Description

I've created a tutorial for migrating from neo4j to memgraph

### Pull request type

Please delete options that are not relevant and check the ones that are.

- [ ] Documentation fix
- [x] New documentation page 
- [ ] Base PR for the release
- [ ] Documentation improvements
- [ ] Other (please describe):

### CODE REVIEW

File: docs/tutorials/migrate-from-neo4j.md [hunk 1]
```diff
@@ -0,0 +1,497 @@
...
+exporting it into Memgraph using [LOAD CSV](/import-data/files/load-csv-clause.md)
+query and Memgraph's user visual interface [Memgraph Lab](/memgraph-lab).
+
+The sample dataset consists of 3 different kinds of nodes (Employee, Order and
+Product) connected with 3 types of relationships as described by the graph
+schema below: 
+
+<img src={require('../data/tutorials/migrate-from-neo4j/shipping_schema.png').default} className={"imgBorder"}/>
+
+TO create this graph in your Neo4j instance run the following queries:
```
Comment (kgolubic) [inline docs/tutorials/migrate-from-neo4j.md:45]: Replace **TO** with **To**.

File: docs/tutorials/migrate-from-neo4j.md [hunk 2]
```diff
@@ -0,0 +1,497 @@
...
+the directory, select the active project, click on *...* -> *Open folder* ->
+*Configuration*. 
+
+Export the data into a CSV file using:
+
+```cypher
+CALL apoc.export.csv.all("shipping.csv", {})
+```
+
+Once exported, the file is located in Ne04j's *Import* folder. To open it,
```
Comment (kgolubic) [inline docs/tutorials/migrate-from-neo4j.md:107]: Fix typo **Ne04j's**.
Is this part necessary in the tutorial " To open it, select the active project, click on ... -> Open folder -> Import."? It seems that it is out of context.

File: docs/tutorials/migrate-from-neo4j.md [hunk 3]
```diff
@@ -0,0 +1,497 @@
...
+
+3. To check if the files are inside the container, first run: 
+
+    ```
+    docker exec -it CONTAINER_ID bash
+    ```
+
+4. List the files inside the `/usr/lib/memgraph`. 
+
+    ```
```
Comment (kgolubic) [inline docs/tutorials/migrate-from-neo4j.md:155]: Replace the start of the code block with ```nocopy so that copy button is not shown.

File: docs/tutorials/migrate-from-neo4j.md [hunk 4]
```diff
@@ -0,0 +1,497 @@
...
+
+But, if you import on one thread, batch of data after a batch of data, there
+should be absolutely no issues, and you will gain 6 times faster import with 6
+times less memory consumption.
+
+After import you can switch back to the `IN_MEMORY_TRANSACTIONAL` storage mode or
+continue running analytics queries (only read queries) in the
+`IN_MEMORY_ANALYTICAL` mode to continue benefiting from low memory consumption. 
+
+**To switch between modes, run the following queries on a running instance:**
```
Comment (kgolubic) [inline docs/tutorials/migrate-from-neo4j.md:202]: Why is this bolded?

File: docs/tutorials/migrate-from-neo4j.md [hunk 5]
```diff
@@ -0,0 +1,497 @@
...
+STORAGE MODE IN_MEMORY_TRANSACTIONAL;
+```
+
+To check the current storage mode, run: 
+
+```cypher
+SHOW STORAGE INFO;
+```
+
+**Change the storage mode to analytical before import.**
```
Comment (kgolubic) [inline docs/tutorials/migrate-from-neo4j.md:215]: Why is this bolded?

File: docs/tutorials/migrate-from-neo4j.md [hunk 6]
```diff
@@ -0,0 +1,497 @@
...
+should be absolutely no issues, and you will gain 6 times faster import with 6
+times less memory consumption.
+
+After import you can switch back to the `IN_MEMORY_TRANSACTIONAL` storage mode or
+continue running analytics queries (only read queries) in the
+`IN_MEMORY_ANALYTICAL` mode to continue benefiting from low memory consumption. 
+
+**To switch between modes, run the following queries on a running instance:**
+
+```cypher
```
Comment (kgolubic) [inline docs/tutorials/migrate-from-neo4j.md:204]: Do you think this block is needed? If we tell users to change the mode to storage, do we need these two Cypher commands here?

File: docs/tutorials/migrate-from-neo4j.md [hunk 7]
```diff
@@ -0,0 +1,497 @@
...
+WITH row WHERE row._labels = ':Employee'
+CREATE (e:Employee {nodeID: row._id})
+```
+
+So `nodeID: row._id` part of the query instructs Memgraph to create a property
+named `nodeID` and assign it the value paired with key `_id`. First created
+node will be assigned the value from the first data row, second node from the
+second data row, etc.
+
+**After matching up the keys and values for all properties, the finished query looks like this:**
```
Comment (kgolubic) [inline docs/tutorials/migrate-from-neo4j.md:275]: Why is this bolded?

File: docs/tutorials/migrate-from-neo4j.md [hunk 8]
```diff
@@ -0,0 +1,497 @@
...
+
+Copy the query in the **Cypher Editor** and **Run Query**.
+
+<img src={require('../data/tutorials/migrate-from-neo4j/employees.png').default} className={"imgBorder"}/>
+
+#### Order nodes
+
+Relevant properties for the `Order` nodes are `_id`, `orderID` and `shipName`.
+
+**To create `Order` nodes run the following query:**
```
Comment (kgolubic) [inline docs/tutorials/migrate-from-neo4j.md:296]: Why is this bolded?

File: docs/tutorials/migrate-from-neo4j.md [hunk 9]
```diff
@@ -0,0 +1,497 @@
...
+
+#### Product nodes
+
+Relevant properties for the `Product` nodes are `_id`, `productID`, `productName`
+and `unitPrice`.
+
+As the parser parses all the values as strings, and the `unitPrice` are numbers,
+they need to be converted to appropriate data type.
+
+**To create `Product` nodes run the following query:**
```
Comment (kgolubic) [inline docs/tutorials/migrate-from-neo4j.md:319]: Why is this bolded?

File: docs/tutorials/migrate-from-neo4j.md [hunk 10]
```diff
@@ -0,0 +1,497 @@
...
+<img src={require('../data/tutorials/migrate-from-neo4j/product.png').default} className={"imgBorder"}/>
+
+### 4. Graph improvements
+
+At this point it would be nice to improve the look of the nodes visually. At the
+moment, nodes in the graph are represented with their labels, but it would be
+more useful if their name attribute was written. 
+
+To adjust the look of the graph with using Graph Style Language, open the the
+Graph Style Editor. On line 22 (or in that vicinity) you will find this block of
```
Comment (kgolubic) [inline docs/tutorials/migrate-from-neo4j.md:341]: I would rephrase this sentence something along the like "look for the following code" so that line number is not mentioned.

File: docs/tutorials/migrate-from-neo4j.md [hunk 11]
```diff
@@ -0,0 +1,497 @@
...
+
+@NodeStyle HasProperty(node, "productName") {
+  label: AsText(Property(node, "productName"))
+}
+```
+
+<img src={require('../data/tutorials/migrate-from-neo4j/GSS.png').default} className={"imgBorder"}/>
+
+Visual appearance of the graph can be changed in many different ways, so be sure
+to check the GSS documentation.
```
Comment (kgolubic) [inline docs/tutorials/migrate-from-neo4j.md:376]: Add link to GSS documentation.

File: docs/tutorials/migrate-from-neo4j.md [hunk 12]
```diff
@@ -0,0 +1,497 @@
...
+LOAD CSV FROM "/usr/lib/memgraph/shipping.csv" WITH HEADER AS row
+WITH row WHERE row._type = 'REPORTS_TO'
+```
+
+Now instruct Memgraph find the nodes with certain IDs in order to create a
+relationship between them. As node IDs are unique we can just define that any
+node with a certain ID is a starting point, and another node with a certain ID
+is the end point of the relationship type `REPORTS_TO`.
+
+**The LOAD CSV query creates 8 `:REPORTS_TO` relationships:**
```
Comment (kgolubic) [inline docs/tutorials/migrate-from-neo4j.md:411]: Why is this bolded?

File: docs/tutorials/migrate-from-neo4j.md [hunk 13]
```diff
@@ -0,0 +1,497 @@
...
+```
+
+The second query returns all the nodes connected with the `REPORTS_TO` type of
+relationship. 
+
+<img src={require('../data/tutorials/migrate-from-neo4j/reports_to.png').default} className={"imgBorder"}/>
+
+#### :SOLD relationships
+
+**The LOAD CSV query creates 830 `:SOLD` relationships:**
```
Comment (kgolubic) [inline docs/tutorials/migrate-from-neo4j.md:430]: Why is this bolded?

File: docs/tutorials/migrate-from-neo4j.md [hunk 14]
```diff
@@ -0,0 +1,497 @@
...
+#### :CONTAINS relationships
+
+This relationship type has properties about the `quantity` of products one order
+contains.
+
+As the parser parses all the values as strings, and the value of this
+relationship property are numbers, they need to be converted to appropriate
+type. 
+
+**The LOAD CSV query creates 2155 `CONTAINS` relationships:**
```
Comment (kgolubic) [inline docs/tutorials/migrate-from-neo4j.md:456]: Why is this bolded?

**Review by kgolubic — CHANGES_REQUESTED** [review]
I've added some comments. Those are all minor stlying fixes.

**Comment by kgolubic** [comment]
We need  to add this tutorial to Tutorial overview page https://memgraph.com/docs/memgraph/tutorials

File: docs/tutorials/migrate-from-neo4j.md (hunk 2 above)
Comment (vpavicic) [reply docs/tutorials/migrate-from-neo4j.md]: I don't understand why it would be out of context?
I'm instructing the user how to find the file they just exported. It's not very logical that it's in the Import folder + they have a shortcut as how to find it in your OS

File: docs/tutorials/migrate-from-neo4j.md (hunk 4 above)
Comment (vpavicic) [reply docs/tutorials/migrate-from-neo4j.md:206]: Answer for all "Why is this bolded?" questions:

With indexes and storage mode it's because there is a lot of theory there as well, so if you just want to DO instead of know WHY, I'm bolding the parts that need to be done

For the rest - the heading h4 is horrible and easily skipped so I wanted to bold important queries necessary to create data

File: docs/tutorials/migrate-from-neo4j.md (hunk 6 above)
Comment (vpavicic) [reply docs/tutorials/migrate-from-neo4j.md:208]: This part is educational, it first explains both storage mode and how to switch between them. ... i unboldeded this and left the commands as a reference guide, then added the analytical mode command as an instruction

**Comment by vpavicic** [comment]
Changed, updated or left comments :)

File: docs/tutorials/migrate-from-neo4j.md (hunk 2 above)
Comment (kgolubic) [reply docs/tutorials/migrate-from-neo4j.md]: Why would one want to open it in Neo4j after the export? This part I don't understand.
